upload-api/tables/subscription.js

```diff
@@ -72,12 +73,39 @@ export function useSubscriptionTable (dynamoDb, tableName) {
         return {}
       } catch (error) {
         const error_ = error instanceof Error && error.message === 'The conditional request failed' ? new ConflictError({
-            message: `Customer ${item.customer} cannot be given a subscription for ${item.provider}: it already has a subscription`
-          }) : error;
+          message: `Customer ${item.customer} cannot be given a subscription for ${item.provider}: it already has a subscription`
+        }) : error;
         throw error_;
       }
     },
 
+    /**
+     * Get a subscription by ID.
+     * 
+     * @param {import('@web3-storage/upload-api').ProviderDID} provider 
+     * @param {string} subscription 
+     * @returns 
+     */
+    get: async (provider, subscription) => {
+      const response = await dynamoDb.send(new GetItemCommand({
+        TableName: tableName,
+        Key: {
+          provider: {
+            S: provider
+          },
+          subscription: {
+            S: subscription
```

Comment:
could we use marshal instead of manually do it?

Reply:
ah, yes!